Formula/libiodbc.rb

```diff
@@ -3,8 +3,7 @@ class Libiodbc < Formula
   homepage "http://www.iodbc.org/dataspace/iodbc/wiki/iODBC/"
   url "https://github.com/openlink/iODBC/archive/v3.52.13.tar.gz"
   sha256 "4bf67fc6d4d237a4db19b292b5dd255ee09a0b2daa4e4058cf3a918bc5102135"
-  # license ["BSD-3-Clause", "LGPL-2.0"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "BSD-3-Clause"
+  license any_of: ["BSD-3-Clause", "LGPL-2.0"]
```

Comment:
```suggestion
  license any_of: ["BSD-3-Clause", "LGPL-2.0-only"]
```

According to the [`LICENSE` file](https://github.com/openlink/iODBC/blob/2e83a40c602a2de38d3847399ac7fe7cd0d4f8ac/LICENSE#L9-L11):
> Note that the only valid version of the LGPL license as far as this
> project is concerned is the original GNU Library General Public License
> Version 2, dated June 1991.